### ISSUE TITLE
Props reference not found

### ISSUE THREAD

**Issue by Tamiyadd** [xinthink/react-native-material-kit#373]
when clicking on `props reference` in readme you get a `404 page not found` on medium :(

**Comment by Crash--** [comment]
Yeah you're right. I didn't maintain the documentation... but you can find it here : https://github.com/xinthink/react-native-material-kit/tree/gh-pages/docs/lib/mdl
